lib/script/common.js

```diff
@@ -509,6 +509,12 @@ exports.digestsByVal = {
   3: 'TAPSCRIPT'
 };
 
+/**
+ * Leaf version is 0xc0 for BIP-342 scripts
+ */
+
+exports.LEAF_VERSION_TAPSCRIPT = 0xc0;
```

Comment:
Ok since we have this here now, let's use it everywhere:

https://github.com/bcoin-org/bcoin/blob/b6825b0d3424a08d80772755b8df57834a112f66/lib/primitives/tx.js#L1964

plus a few places in taproot-check-test.js